src/extras/geometries/EdgesGeometry.js

```diff
@@ -86,6 +86,8 @@ THREE.EdgesGeometry = function ( geometry, thresholdAngle ) {
 
 	}
 
+	coords = coords.subarray( 0, index );
```

Comment:
Should be: 
```
coords = new Float32Array( coords.subarray( 0, index ) );
```
Else the underlying ```ArrayBuffer``` behind the coords will still be the full allocation size.